build-tools/docker/slim.Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Builder image
+FROM ubuntu:mantic-20231011 as builder
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ARG USERNAME=compositor
+ARG RUST_VERSION=1.74
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -y -qq && \
+  apt-get install -y \
+    build-essential curl pkg-config libssl-dev libclang-dev git sudo \
+    libegl1-mesa-dev libgl1-mesa-dri libxcb-xfixes0-dev mesa-vulkan-drivers \
+    ffmpeg libavcodec-dev libavformat-dev libavfilter-dev libavdevice-dev && \
+  rm -rf /var/lib/apt/lists/*
+
+RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
+RUN source ~/.cargo/env && rustup install $RUST_VERSION && rustup default $RUST_VERSION
+
+COPY . /root/project
+WORKDIR /root/project
+
+RUN source ~/.cargo/env && cargo build --release --no-default-features
+
+# Runtime image
+FROM ubuntu:mantic-20231011
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ARG USERNAME=compositor
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV NVIDIA_DRIVER_CAPABILITIES compute,graphics,utility
```

Comment:
Shouldn't this be?
```suggestion
ENV NVIDIA_DRIVER_CAPABILITIES=compute,graphics,utility
```

Reply:
this was copy paste from original dockerfile
as far as I remember both syntax are valid, but yest we should be consitent